stories/resizable-card-layout/resizable-card-layout.stories.js

```diff
@@ -0,0 +1,179 @@
+export default {
```

Comment:
Let's add there more complex example with container having flexible-cards.
my proposition:
```
<div class="fd-flexible-card-layout"> <-- this can have `flex` with `row-wrap` flow 
    <div class=fd-flexible-card-layout__wrapper or something similar. styles="height: x; width: x">
    <div class=fd-flexible-card-layout__wrapper or something similar. styles="height: x; width: x">
    ...
</div>
```